components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/actions/ExtendWorkspaceTimeoutAction.kt

```diff
@@ -31,10 +31,10 @@ class ExtendWorkspaceTimeoutAction : AnAction() {
                     thisLogger().error("gitpod: failed to extend workspace timeout", e)
```

Comment:
**Thought:** Interesting, I didn't realise this logic was copied into our JetBrains codebase, I assumed it was running the `gp` command under the hood.